media_driver/linux/common/codec/ddi/media_ddi_decode_base.cpp

```diff
@@ -70,6 +70,15 @@ VAStatus DdiMediaDecode::BasicInit(
         return VA_STATUS_ERROR_ALLOCATION_FAILED;
     }
 
+    m_ddiDecodeCtx->pRTtbl = MOS_New(DDI_CODEC_RENDER_TARGET_TABLE);
+
+    if (m_ddiDecodeCtx->pRTtbl == nullptr)
+    {
+        MOS_Delete(m_ddiDecodeCtx->pRTtbl);
+        m_ddiDecodeCtx->pRTtbl = nullptr;
+        return VA_STATUS_ERROR_ALLOCATION_FAILED;
```

Comment:
Why we need delete nullptr?

Reply:
Copy-pasted for consistency. Lines 77 and 78 have no effect, really, will remove.